responsibleai_text/responsibleai_text/utils/genai_metrics/scripts/coherence.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+"""Coherence metric."""
+
+import datasets
+import evaluate
+import pandas as pd
+
+logger = evaluate.logging.get_logger(__name__)
+
+
+_CITATION = """
```

Comment:
can we move _CITATION and _KWARGS_DESCRIPTION  to a common constants file and re-use for all metrics?